Cue/app/tabs/library/LibraryHome.js

```diff
@@ -66,10 +66,12 @@ class LibraryHome extends React.Component {
 
   constructor(props: Props) {
     super(props)
-
+
+    let refreshing = this.props.decks ? false : true;
+
     this.state = {
       editing: false,
-      refreshing: false,
+      refreshing: refreshing,

```

Comment:
You don't need to write `refreshing: refreshing`, you can just write `refreshing`.